### PR TITLE
fix(release): fix release action divergence

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

* fix minor issues and divergence causing release action to fail when creating tags and pushing to jira
* align get-version workflow case with develop

Fixes #MON-113562

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] master

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: .github/workflows/get-version.yml [hunk 1]
```diff
@@ -156,10 +161,20 @@ jobs:
...
+
+              # Check if pull request branch targeting develop
+              if [[ "$GITHUB_BRANCH_BASE_REF_NAME" == "develop" ]] && [[ "$GITHUB_BRANCH_PR_STATE" == "OPEN" ]]; then
+                echo "release_cloud=1" >> $GITHUB_OUTPUT
+                echo "release_type=$GITHUB_RELEASE_TYPE" >> $GITHUB_OUTPUT
+              else
+                echo "release_cloud=$GITHUB_RELEASE_CLOUD" >> $GITHUB_OUTPUT
+                echo "release_type=$GITHUB_RELEASE_TYPE" >> $GITHUB_OUTPUT
+              fi
+              ;;              ;;
```
Comment (mushroomempires) [inline .github/workflows/get-version.yml:177]:
```suggestion
              ;;
```

File: .github/workflows/get-version.yml [hunk 2]
```diff
@@ -156,10 +161,20 @@ jobs:
...
             *)
+              # Any branch name
+              GITHUB_BRANCH_BASE_REF_NAME="$(gh pr view $BRANCHNAME -q .baseRefName --json headRefName,baseRefName,state)"
+              GITHUB_BRANCH_PR_STATE="$(gh pr view $BRANCHNAME -q .state --json headRefName,baseRefName,state)"
               echo "release=`date +%s`.`echo ${{ github.sha }} | cut -c -7`" >> $GITHUB_OUTPUT
-              echo "release_cloud=$GITHUB_RELEASE_CLOUD" >> $GITHUB_OUTPUT
-              echo "release_type=$GITHUB_RELEASE_TYPE" >> $GITHUB_OUTPUT
-              ;;
+
+              # Check if pull request branch targeting develop
```
Comment (mushroomempires) [inline .github/workflows/get-version.yml:169]:
```suggestion
              # Check if the pull request is targeting develop
```

Comment (mushroomempires) [reply .github/workflows/get-version.yml:169]: (should probably done in every PR if it's not too much of a hassle since it's really minor)

Comment (tuntoja) [reply .github/workflows/get-version.yml:169]: will handle it separately to avoid remove all reviews

**Pull request merged**